src/main/webapp/user_profile_style.css

```diff
@@ -155,4 +155,40 @@ th, td {
 
 .delete-task:hover i {
 	color: white;
+}
+
+#createTaskModal {
+    display: none; /* Hidden by default */
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    padding-top: 100px; /* Location of the box */
+    left: 0;
+    top: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+#modal {
+    background-color: #fefefe;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 80%;
```

Comment:
This seems a bit too wide for me, considering the size of the text area inputs. I personally would make this a bit narrower and/or make the task details input bigger, but that's a personal opinion so either way works!

Reply:
I modified the style for the modal box and now it should look better.